### PR TITLE
[DEV APPROVED] 10156 Add the google tag manager script to the not_found page

### PR DESCRIPTION
**Pull request by margOnline**
[TP 10156](https://moneyadviceservice.tpondemand.com/restui/board.aspx?#page=board/5682182231901749200&appConfig=eyJhY2lkIjoiRkRBMzk3RTBGNjU3NDg1RDM4QzczMTJFNzI5MDFBN0MifQ==&searchPopup=userstory/10156)

Display the google tag manager on the not found page.

### CODE REVIEW

**Comment by masbuild** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There are code changes, but no corresponding tests. Please include tests if this PR introduces any modifications in behavior. If this is a refactor PR, please ignore the message.Refer to our mas standards guides about opening a PR<a href="link-to-the-mas-standards-about-opening-a-pr">here</a></td>
    </tr>
  </tbody>
</table>

At Money Advice Service, we use Danger gem, to ensure that our commits and pull requests follow [standards](https://github.com/moneyadviceservice/mas-standards/blob/master/Dangerfile). If you think one of the points here has been raised in error (or shouldn't be in our standards), open an issue or pull request on [our Danger config](https://github.com/moneyadviceservice/mas-standards).

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -27,22 +27,7 @@
...
-            w[l] = w[l] || [];
-            w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
-            var f = d.getElementsByTagName(s)[0],
-                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : '';
-            j.async = true;
-            j.src = '//www.googletagmanager.com/gtm.js?id=' + i + dl;
-            f.parentNode.insertBefore(j, f.nextSibling);
-        })(window, document, 'script', 'dataLayer', '<%= Rails.application.config.google_tag_manager_id %>');
-    </script>
+        <%= render 'shared/google_tag_manager' %>
```
Comment (MarcoBgn) [inline app/views/layouts/application.html.erb]: 👍

**Comment by margOnline** [comment]
@MarcoBgn its failing before that. Also the the brakeman gem is not locked to any version in the gemfile.

**Comment by margOnline** [comment]
@MarcoBgn ♻️

**Pull request merged**